README.md

```diff
@@ -67,36 +67,54 @@ Note that as of November 6, 2023, OpenShift Serverless Operator is based on RHEL
 
-2. Create a namespace for the Orchestrator solution:
+2. Deploy the PostgreSQL reference implementation for persistence support in SonataFlow following these [instructions](https://github.com/parodos-dev/orchestrator-helm-chart/blob/gh-pages/postgresql/README.md)
+
+3. Create a namespace for the Orchestrator solution:
 
    ```console
    oc new-project orchestrator
    ```
 
-3. Set GITHUB_TOKEN environment variable
+4. Set `GITHUB_TOKEN` environment variable
 
    ```console
    export GITHUB_TOKEN=<github token>
    ```
 
-4. Deploy PostgreSQL reference implementation following these [instructions](https://github.com/parodos-dev/orchestrator-helm-chart/blob/gh-pages/postgresql/README.md)
+5. Create a namespace for the Red Hat Developer Hub Operator (RHDH Operator):
 
-### ...without GitOps
+   ```console
+   oc new-project rhdh-operator
+   ```
 
-1.  Install the orchestrator Helm chart:
+6. Generate a random alphanumeric string of 24 characters to be used in the `BACKEND_SECRET`:
 
-    ```console
-    helm upgrade -i orchestrator orchestrator --set rhdhOperator.github.token=$GITHUB_TOKEN
-    ```
+   ```console
+   ewxport BACKEND_SECRET=$(mktemp -u XXXXXXXXXXXXXXXXXXXXXXX)
+   ```
+
+###  Deployment without GitOps
+
+7. Create the secret in the 'rhdh-operator' namespace with the GitHub token:
+
+   ```console
+   oc create secret generic backstage-backend-auth-secret --from-literal=BACKEND_SECRET=$BACKEND_SECRET --from-literal=GITHUB_TOKEN=$GITHUB_TOKEN
+   ```
+
+   Install the orchestrator Helm chart:
+
+   ```console
+   helm upgrade -i orchestrator orchestrator
+   ```
 
-2.  Run the commands prompted at the end of the previous step to wait until the services are ready.
+   Run the commands prompted at the end of the previous step to wait until the services are ready.
 
-### ... with GitOps
+### Deployment with GitOps
 
-1.  Install `Red Hat OpenShift Pipelines` and `Red Hat OpenShift GitOps` operators following these [instructions](https://github.com/parodos-dev/orchestrator-helm-chart/blob/gh-pages/gitops/README.md).
+7.  Install `Red Hat OpenShift Pipelines` and `Red Hat OpenShift GitOps` operators following these [instructions](https://github.com/parodos-dev/orchestrator-helm-chart/blob/gh-pages/gitops/README.md).
     The Orchestrator installs RHDH and imports software templates designed for bootstrapping workflow development. These templates are crafted to ease the development lifecycle, including a Tekton pipeline to build workflow images and generate workflow K8s custom resources. Furthermore, ArgoCD is utilized to monitor any changes made to the workflow repository and to automatically trigger the Tekton pipelines as needed. This installation process ensures that all necessary Tekton and ArgoCD resources are provisioned within the same cluster.
 
-2.  Run the following command to set up environment variables:
+8.  Run the following command to set up environment variables:
 
     ```console
     ./hack/setenv.sh --use-default
```

Comment:
It seems this script is mainly no longer relevant and can be removed from the instruction.

Reply:
I suggest to rename the script and keep it for creating the mandatory secret.